content/pandas/concepts/dataframe/terms/resetindex/reset-index.md

```diff
@@ -0,0 +1,168 @@
+---
+Title: '.reset_index()'
+Description: 'This article is about how the .reset_index() is used in pandas dataframes.'
+Subjects: 
+  - 'Data Science'
+  - 'Pandas'
+Tags:
+  - 'Methods'
+  - 'Pandas'
+  - 'Data Science'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
+This article is about how the **.reset_index()** is used in pandas dataframes.
+
+
+## Syntax
+```py
+import pandas as pd
+df.read_csv('some_csv.csv', index_col = 'Account Number')
+df.reset_index()
+```
```

Comment:
There should be a paragraph or two after the syntax block explaining the parameters.